.github/workflows/protocol_tests.yml

```diff
@@ -39,8 +39,6 @@ jobs:
 
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
-        with:
```

Comment:
Is there any particular reason for this change ?

Reply:
This is to keep up to date with foundry. 

We have made the mistake in the past to specify a specific version, but that tends to not get updated, and when it finally does it's a bigger task.

Reply:
(for future reference)

I ran into a limitation with this in this PR: 
- https://github.com/celo-org/celo-monorepo/pull/11002

The latest version has a regression that leads to a failing integration test. 
But, fixing a previous nightly build fixed the error.

More context [here](https://github.com/celo-org/celo-monorepo/pull/11002/files#r1624810719).